### DISCUSSION TITLE
Tailwind UI like UI library for Bootstrap - Ayro UI

### DISCUSSION THREAD

**Discussion by Musharofchy** [twbs/bootstrap#35385] [Show & Tell]
Hey Bootstrap Community,

I am Musharof from [Ayro UI](https://ayroui.com/), me with my team built Tailwind UI like solution for Bootstrap.

Ayro UI is a UI library that provides Bootstrap UI Components Snippets for Creating Fast & Responsive - Landing Pages, Templates, or Websites by Copy-Pasting Snippets. We added tons of new UI components, crafted a well-thought style guide with easy to use copy-paste UI Builder, and made all core components free and open-source.

GitHub: https://github.com/ayroui/free-ui-components

Website: https://ayroui.com/

If you have any feedback or thoughts please, let me know by comments. I am posting here not only for promotion but, literally wants to improve the library and would like to hear from lovely Bootstrap community.

Have a good day
Musharof

**Comment by ErikThiart** [comment]
I am glad I found this. Is this still an active project? 
I am interested in buying the lifetime license, if this is still an active project. 

Edit: Looks abandoned, last GitHub updates was >3 years ago.
